### ISSUE TITLE
No toolbar keyboard focus states in high contrast mode

### ISSUE THREAD

**Issue by slimjim181** [mozilla/pdf.js#7577]
As a visually impaired user
I want to be able to navigate the toolbar of pdf.js*
So I can perform actions on my PDF documents

PDF file is not applicable but screenshots were taken with the example pdf https://mozilla.github.io/pdf.js/web/viewer.html on a Windows 10 IE11

**Configuration:**
-  Internet Explorer 11 (11.545.10586.0 latest) most likely the case in earlier version of IE too
-  Windows 7, 8 and 10 in high contrast mode 1
-  PDF.js version: 0.8.0 (latest)

**Steps to reproduce the problem:**
1. Open up PDF.js in IE11 on Windows 7, 8 or 10
2. Change the windows theme to high contrast 1
3. Try to use the keyboard to select one of the toolbar buttons (tab through)

**Expected behaviour:**
A focus state is visible on the current toolbar button I have focused
![ie11 win10 pdfjs high contrast no focus states 2](https://cloud.githubusercontent.com/assets/20076864/18073793/9de619cc-6ebc-11e6-9671-20bed6aae9ca.jpg)

**Actual behaviour:**
There are no focus states
Note:
- Win 8 and 10 has a pop-up tooltip that appears about 1 second after keyboard focusing a button. Although this is helpful this is not an actual focus state\* 
  ![ie11 win10 pdfjs high contrast no focus states](https://cloud.githubusercontent.com/assets/20076864/18073799/ab8c0c9e-6ebc-11e6-8ee7-05c290ed2c29.jpg)
- Win 7 does not show these pop-ups _at all_ so it is impossible for a visually impaired user to navigate this page.

*See https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

**Comment by yurydelendik** [comment]
Works as expected in Firefox.

**Comment by timvandermeij** [comment]
Closing IE11/Edge (non-Chromium-based) issues in response to #11211. Please note that, as outlined in https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support, only bugs which completely prevent the library and/or the default viewer from running will be accepted.
